### ISSUE TITLE
The pycbc install script ask if you want the ROM data twice

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#803]
The current version of the install script asked about the ROM data twice. It should just ask once.

```
To run MKL optimized code, you need to enter the path and architecture
for the Intel optimized toolkit on your cluster. For example:
on sugar, enter
     /opt/intel/bin/compilervars.sh intel64
on atlas, enter
     /opt/intel/2015/bin/compilervars.sh intel64
on ldas-grid, enter
     /opt/intel/composer_xe_2015.1.135/bin/compilervars.sh intel64
If you do not have these tools installed, just press return.

Enter path and architecture for Intel compilervars.sh or press return: /opt/intel/composer_xe_2015.1.135/bin/compilervars.sh intel64

Is the LAL Reduce Order Model (ROM) data installed on your cluster (Enter yes or no, if unsure type no)? no

Do you want to download the ROM data now (Enter yes or no): yes

Enter the path to store the ROM data: ~/src/lal-data             

Is the LAL Reduce Order Model (ROM) data installed on your cluster (Enter yes or no, if unsure type no)? no

Do you want to download the ROM data now (Enter yes or no): yes

Enter the path to store the ROM data: ~/src/lal-data

------------PLease check the inputs carefully-----------------

LIGO.ORG username:  duncan.brown
Lalsuite Branch or Tag:  o1_lalinference_20160326
Development or released:  2
   (1 is release, 2 is development)
Github Username:  duncan-brown
Path and architecture for Intel compilervars.sh: /opt/intel/composer_xe_2015.1.135/bin/compilervars.sh intel64
ROM data will be install in /home/dbrown/src/lal-data
```

**Comment by lenona** [comment]
Addressed in #812

**Comment by duncan-brown** [comment]
Fixed.
